pkgs/by-name/co/coulr/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  wrapGAppsHook4,
+  pkg-config,
+  meson,
+  ninja,
+  appstream-glib,
+  desktop-file-utils,
+  gobject-introspection,
+  libadwaita,
+  libportal-gtk4,
+  libnotify,
+  nix-update-script,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "coulr";
+  version = "2.1.0";
+
+  format = "other";
+  dontWrapGApps = true;
+
+  src = fetchFromGitHub {
+    owner = "Huluti";
+    repo = "Coulr";
+    rev = version;
+    hash = "sha256-1xnL5AWl/rLQu3i9m6uxbS4QT+690hmEW8kYTwkg7Gw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    appstream-glib
+    desktop-file-utils
+    gobject-introspection
+    wrapGAppsHook4
+  ];
+
+  propagatedBuildInputs = [
+    python3.pkgs.pygobject3
+    libadwaita
+    libportal-gtk4
+    libnotify
+  ];
+
+  postPatch = ''
+    patchShebangs build-aux/meson/postinstall.py
+    substituteInPlace build-aux/meson/postinstall.py --replace gtk-update-icon-cache gtk4-update-icon-cache
```

Comment:
```suggestion
    substituteInPlace build-aux/meson/postinstall.py \
      --replace-fail gtk-update-icon-cache gtk4-update-icon-cache
```

Reply:
See https://github.com/NixOS/nixpkgs/issues/302605. This is fine, but better to open a PR upstream afterwards.